src/main.c

```diff
@@ -880,10 +880,48 @@ ubus_poe_reload_cb(struct ubus_context *ctx, struct ubus_object *obj,
 	return UBUS_STATUS_OK;
 }
 
+static const struct blobmsg_policy ubus_poe_manage_policy[] = {
+	{ "port", BLOBMSG_TYPE_STRING },
+	{ "enable", BLOBMSG_TYPE_BOOL },
+};
+
+static int ubus_poe_manage_cb(struct ubus_context *ctx,
+			      struct ubus_object *obj,
+			      struct ubus_request_data *req,
+			      const char *method, struct blob_attr *msg)
```

Comment:
I think you can just about fit (ctx, obj), and (req, method) on the same line.

(optional) If you don't use an argument, feel free to leave the name out. For example:
```
static int ubus_poe_manage_cb(struct ubus_context *, struct ubus_object *,
			      struct ubus_request_data *, const char *,
			      struct blob_attr *msg)

```